.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  lockfile-lint:
+    name: Lockfile lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: lint lock file
+        run: npx lockfile-lint --path package-lock.json --allowed-hosts npm --validate-https
+
+  test:
+    strategy:
+      matrix:
+        platform: [ ubuntu-latest ]
+        node: [ '10', '12', '14' ]
+    needs: lockfile-lint
+    name: Unit Tests Node ${{ matrix.node }} (${{ matrix.platform }})
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Comment:
Could you update to 2.1.5?

Reply:
Looking at the tags and releases it seems that v2 is always updated so I'm not sure it's needed https://github.com/actions/setup-node/tags